Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json

```diff
@@ -0,0 +1,162 @@
+{
+    "id": "AIVectraStream",
+    "title": "AI Vectra Stream",
+    "publisher": "Vectra AI",
+    "descriptionMarkdown": "The AI Vectra Stream connector allows to send Network Metadata colelcted by Vectra Sensors accross the Network and Cloud to Azure Sentinel", 
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Install Kusto functions prior creating new queries or workbook.",
```

Comment:
Parser link should be the part of additional requirement banner description. Please add the parser link in this format https://aka.ms/sentinel-Parser-AIVectraStream.

Thanks,
Aditi Maheshwari

Reply:
@v-admahe, how those links are built? Any pointer?